### PR TITLE
Rename synopsis::Header to synopsis::Synopsis #69

### PR DESCRIPTION
**Pull request by ruspl-afed**
Change synopsis.ecore and related resources
Change markup.ecore and related resources

Signed-off-by: Alexander Fedorov <alexander.fedorov@arsysop.ru>

### CODE REVIEW

File: bundles/ru.arsysop.loft.rgm.markup.ecore/model/markup.ecore [hunk 1]
```diff
@@ -4,8 +4,8 @@
   <eClassifiers xsi:type="ecore:EClass" name="Markup">
     <eStructuralFeatures xsi:type="ecore:EReference" name="citations" upperBound="-1"
         eType="#//Citation" containment="true"/>
-    <eStructuralFeatures xsi:type="ecore:EReference" name="headers" upperBound="-1"
-        eType="ecore:EClass ../../ru.arsysop.loft.rgm.synopsis.ecore/model/synopsis.ecore#//Header"/>
+    <eStructuralFeatures xsi:type="ecore:EReference" name="synopses" upperBound="-1"
```
Comment (zelenyhleb) [inline bundles/ru.arsysop.loft.rgm.markup.ecore/model/markup.ecore:7]: synopses?

**Review by zelenyhleb — COMMENTED** [review]
Is synopses the right name, @ruspl-afed ?

File: bundles/ru.arsysop.loft.rgm.markup.ecore/model/markup.ecore (hunk 1 above)
Comment (ruspl-afed) [reply bundles/ru.arsysop.loft.rgm.markup.ecore/model/markup.ecore:7]: They say this is plural form

**Comment by ruspl-afed** [comment]
> Is synopses a right name, @ruspl-afed ?

They say this is plural form for `synopsis` @zelenyhleb

**Comment by zelenyhleb** [comment]
> > Is synopses a right name, @ruspl-afed ?
> 
> They say this is plural form for `synopsis` @zelenyhleb 

ok

**Review by zelenyhleb — APPROVED** [review]

**Pull request merged**